ng/manage.ts

```diff
@@ -436,6 +436,72 @@ angular.module("armExplorer", ["ngRoute", "ngAnimate", "ngSanitize", "ui.bootstr
         window.location.href = "api/tenants/" + $scope.selectedTenant.id;
     };
 
+    $scope.resourceSearch = () => {
+        // trigger a refresh if needed, but do not need to wait for it to finish since cache should already exist
+        refreshResourceSearchCache();
+
+        var results: IResourceSearchSuggestion[] = [];
+        var keyword = $scope.resourceSearchModel.searchKeyword || "";
+        var stringCompare = (strA: string, strB: string): number=> {
```

Comment:
why do you need to define your own string compare? can't you just use `.localeCompare()`